packages/header-change-detection/lib/lambda/header-check.ts

```diff
@@ -0,0 +1,300 @@
+import axios from "axios";
+import { DynamoDBClient, BatchGetItemCommand, BatchGetItemCommandInput, KeysAndAttributes, UpdateItemCommandInput, UpdateItemCommand, AttributeValueUpdate } from "@aws-sdk/client-dynamodb";
+import { PublishCommand, PublishInput, SNSClient } from "@aws-sdk/client-sns";
+
+const URLS = process.env.URLS;
+const HEADERS = process.env.HEADERS;
+const TABLE = process.env.TABLE!;
```

Comment:
We've been playing around in a couple of projects with using [env-var](https://github.com/evanshortiss/env-var) for erroring nicely when required environment variables are missed. Might be something to consider here, not essential.

Reply:
I generally don't mind when passing stuff into Lambdas because unless something has gone terribly wrong it will always have a value passed in through the build process. It would be _better_ to have something to actually check it but I don't think it's that important here